Casks/font-ibm-plex-mono.rb

```diff
@@ -2,24 +2,26 @@
   version :latest
   sha256 :no_check
 
-  url "https://github.com/google/fonts/trunk/ofl/ibmplexmono",
-      verified: "github.com/google/fonts/",
-      using:    :svn
+  url "https://github.com/google/fonts",
+      verified:   "github.com/google/fonts",
+      branch:     "main",
+      only_paths: ["ofl/ibmplexmono"],
+      using:      :git
```

Comment:
```suggestion
      only_paths: ["ofl/ibmplexmono"]
```

This might even work (or omitting just one of them). No idea but might be worth playing with.

Reply:
Ah, good point! I saw that the download strategy detector was choosing the `GitHubGitDownloadStrategy`, but I hadn't registered that it's a subclass of `GitDownloadStrategy`. I confirmed that removing `using: :git` and instead appending `.git` to the URL does work 👍 

I tried removing `branch`, but it seems like we assume `master` is the default branch:

```
==> Cloning https://github.com/google/fonts.git
Cloning into '/Users/hmarr/Library/Caches/Homebrew/Cask/font-ibm-plex-mono--git'...
fatal: Remote branch master not found in upstream origin
Error: Download failed on Cask 'font-ibm-plex-mono' with message: Failure while executing; `git clone --branch master --no-checkout --filter=blob:none -c advice.detachedHead=false https://github.com/google/fonts.git /Users/hmarr/Library/Caches/Homebrew/Cask/font-ibm-plex-mono--git` exited with 128. Here's the output:
Cloning into '/Users/hmarr/Library/Caches/Homebrew/Cask/font-ibm-plex-mono--git'...
fatal: Remote branch master not found in upstream origin
```

Reply:
> I tried removing branch, but it seems like we assume master is the default branch:

Ah, cool, all good, thanks!